lib/database.js

```diff
@@ -164,6 +166,21 @@ function remainingXp (totalXp) {
   return 100 - totalXp % 100
 }
 
+
```

Comment:
Please change the upVote/downVote functions to use the following format.

function upVote (id, video_id) {
  return knex ('videos')
    .where('id', video_id)
    .increment('votes', 1)
    .then(() => {
      return getSkillsById (id)
    })
}